assets/dora/man_route_opt_lb.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Can you please rename the manifest to include the word "manifest". There are many other manifests in CATs and typically they all have "manifest" in their filename to make it easy to search for them all.

How's `route_options_manifest.yml`?